src/OrchardCore.Modules/OrchardCore.AuditTrail/Models/AuditTrailEvent.cs

```diff
@@ -0,0 +1,58 @@
+﻿using OrchardCore.Entities;
+using System;
+
+namespace OrchardCore.AuditTrail.Models
+{
+    public class AuditTrailEvent : Entity
+    {
+        /// <summary>
+        /// The ID of the event.
+        /// </summary>
+        public string Id { get; set; }
```

Comment:
That's a good point! Seems strange, like a leftover from a content item index. @domonkosgabor ?

Reply:
It would probably be sensible to include a guid id (`IIdGenerator`) here @domonkosgabor i.e. `AuditTrailEventId` rather than using a database Id.

A guid will be necessary / useful if / when this gets extended to support audit trails across deployments, i.e. stage / prod servers where you want to include an audit history of content that is deployed through deployment plans.

It may not be necessary right now, but not having it from the get go will make extending support harder in the future.

Reply:
@domonkosgabor this isn't addressed.